src/pages/updates/work-in-progress/index.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Work in progress
+description:
+  This page provides early visibility for new Carbon concepts and current work
+  in progress.
+---
+
+<PageDescription>
+
+An opportunity to see early Carbon concepts and ideas, and experiment with work
+in progress.
+
+</PageDescription>
+
+<InlineNotification>
+  This work in progress is not actively supported by the Carbon team, and is not
+  approved for production use. We gratefully accept feedback or comment.
+</InlineNotification>
+
+## Introduction
+
+The Carbon team is always exploring different ways to serve our users. We work
+in the open, and this page is here to give you early previews of what we're
+working on.
+
+That could mean making an unfinished Sketch kit available for download, linking
+to GitHub issues for proposed components and patterns, or sharing graphics files
+of concepts for how we might present content.
+
+Not everything here may make it into production, while other pieces might become
+central pieces of the design system. It's an early opportunity for you to
+provide feedback and have a say in Carbon's direction.
+
+### Tools and resources
+
+<CardGroup>
+  <MiniCard
+    onClick={() => fathom('trackGoal', 'GC8C8OMU', 0)}
+    title="Carbon mobile sketch kit"
+    href="https://www.sketch.com/s/a3343128-adb6-489c-9e62-709d89ba76e9"
+    actionIcon="launch"
+  />
+  <MiniCard
+    onClick={() => fathom('trackGoal', 'GP38VQMY', 0)}
+    title="Data viz sticker sheet"
+    href="https://ibm.box.com/s/epx74uadotxbx36aknn1s2nmvhpzgk4o"
```

Comment:
So it says it linking to the box file but when I click on the tile its taking me to the mobile sketch kit instead of the data viz file. Is this happening for anyone else?

Reply:
This should be working OK now. Build of the push that fixed it was delayed.